ManageCoursesUi.Tests/ViewModelHelperTests.cs

```diff
@@ -86,22 +87,33 @@ public void TestGetSiteStatus(string status, string expectedResult)
         }
 
         [Test]
-        [TestCase("d", true, "")]
-        [TestCase("d", false, "")]
-        [TestCase("s", true, "")]
-        [TestCase("s", false, "")]
-        [TestCase("b", true, "Yes")]
-        [TestCase("b", false, "No")]
-        public void TestGetHasVacancies(string status, bool hasVacancies, string expectedResult)
+        [TestCase("d", "B", "")]
+        [TestCase("d", "F", "")]
+        [TestCase("d", "P", "")]
+        [TestCase("s", "B", "")]
+        [TestCase("s", "F", "")]
+        [TestCase("s", "P", "")]
+
+        [TestCase("", "", "No")]
+        [TestCase("b", "B", "Yes")]
+        [TestCase("b", "F", "Yes")]
+        [TestCase("b", "P", "Yes")]
+        [TestCase("b", "", "No")]
+
+        // These are probably wrong
+        [TestCase("", "B", "Yes")]
+        [TestCase("", "F", "Yes")]
+        [TestCase("", "P", "Yes")]
+        public void TestGetHasVacancies(string status, string vacStatus, string expectedResult)
```

Comment:
how come the change of type of vacancy status got tied up with this PR?

Reply:
The test vs the approach to make it pass changed as the underlying type has changed, vac was in master

Reply:
boy scout rulez appiled